packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/data/TerariumAssetServiceWithSearch.java

```diff
@@ -364,4 +380,39 @@ public void updateIndexInPlace() throws IOException {
 		// refresh the index
 		elasticService.refreshIndex(getAssetAlias());
 	}
+
+	protected boolean isRunningTestProfile() {
```

Comment:
I'm starting to see this method copied and pasted around more and more (I just had to use it for checking test with the DKG import). I wonder if we should centralize this....